tests/utils/managed_deployment.py

```
@@ -37,6 +37,65 @@ def _get_workspace_dir() -> str:
+def _api_connection_errors() -> tuple[type[BaseException], ...]:
+    # ConnectionError/ConnectionRefusedError/TimeoutError and
+    # aiohttp.ClientConnectorError (kubernetes_asyncio) are all OSError subclasses;
+    # httpx.TransportError (kr8s) is not, so add it explicitly. Built defensively
+    # so a missing optional client library never breaks import.
+    errs: list[type[BaseException]] = [OSError]
+    try:
+        import httpx
+
+        errs.append(httpx.TransportError)
+    except Exception:
+        pass
```

Ruff (0.16.1)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)